docs/source/es/attention.md

```diff
@@ -0,0 +1,41 @@
+<!--Copyright 2023 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+
+⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+
+-->
+
+# Mecanismos de atención
+
+La mayoría de los modelos de transformadores utilizan atención completa, en el sentido de que la matriz de atención es cuadrada. Esto puede ser un gran cuello de botella computacional cuando tienes textos largos. `Longformer` y `reformer` son modelos que intentan ser más eficientes y utilizan una versión dispersa de la matriz de atención para acelerar el entrenamiento.
```

Comment:
When writing about transformers (specifically in the phrase "modelos de transformadores"), I believe that using 'transformadores' may not be the most accurate translation. In my opinion, it is preferable to consistently refer to them as "transformers," regardless of the language being used.

Reply:
@aaronjimv @stevhliu I just added this comment. I'm not sure if it's correct, but I wanted to express my thoughts on this.

Reply:
hi @tadeodonegana thanks, I thinks its a good point. I going to use to "`transformers`"

Reply:
I think when referring to the library, it is good to refer to it as just Transformers (no translation), but when referring to the models in a more generic way then it is ok to translate it.

Reply:
Sounds good to me! Happy to help reviewing future translations! Great work @aaronjimv!